libkbfs/block_journal.go

```diff
@@ -254,6 +279,8 @@ func (j *blockJournal) readJournal(ctx context.Context) (
 			}
 		}
 	}
+	j.log.CDebugf(ctx, "Found %d block bytes in the journal", blockBytes)
+	j.blockBytes = blockBytes
```

Comment:
`shutdown` doesn't check the value.  The original problem was that I didn't have an exception above for `os.IsNotExist(err)` and instead just returned that error, which caused `shutdown` to panic.  But I ended up having to add an exception in that case anyway because the DSL tests can end up removing block data (via `BlockServerDisk`) without removing the corresponding journal entry.  I'm still happy to return the value here though.